examples/counter-example/.env.example

```
@@ -1,3 +1,9 @@
-MH_ACCOUNT=your_account_address_here
-MH_NETWORK=testnet
\ No newline at end of file
+# Private Key (used for all networks - Hardhat-style)
+# This is your wallet's private key that will be used to deploy and interact with contracts
+PRIVATE_KEY=your_private_key_here
+
+# Optional: Override RPC URL (useful for custom endpoints)
+# MOVEMENT_RPC_URL=https://custom-testnet.movementnetwork.xyz/v1
+
+# Optional: Override default network from config
+# MH_DEFAULT_NETWORK=testnet
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)